process/process_areas/quality_management/guidance/quality_report_template.rst

```diff
@@ -22,43 +22,39 @@ Template Quality Report
    :status: valid
    :complies: std_req__iso26262__management_5423, std_req__aspice_40__SUP-1-BP1, std_req__aspice_40__SUP-1-BP2, std_req__aspice_40__SUP-1-BP3, std_req__aspice_40__SUP-1-BP4, std_req__aspice_40__SUP-1-BP7, std_req__aspice_40__PIM-3-BP1, std_req__aspice_40__PIM-3-BP2, std_req__aspice_40__PIM-3-BP3, std_req__aspice_40__PIM-3-BP4, std_req__aspice_40__PIM-3-BP5, std_req__aspice_40__PIM-3-BP6, std_req__aspice_40__PIM-3-BP7
 
-   This document implements :need:`wp__qms_report`.
+   This document implements :need:`wp__qms_report` and based on the :need:`wp__qms_plan`.
+
+   The Quality Report contains:
+
+   **1. Quality Overview**
+
+   **1.1. Lists of all work products**
+    - List of all work products, compared to the :need:`wp__qms_plan`
+    - Status (valid / invalid)
+
+   **1.2. Lists of all reports**
+    - List of all reports, compared to the :need:`wp__qms_plan`
+    - Results of the reports
+
+   **1.3. Test coverage**
+    - Overview of test coverage (overall, requirements, architecture)
+    - Ratio of test coverage to the :need:`wp__verification_plan`
+
+   **1.4. Issues**
+    - List of all issues, compared to the :need:`wp__qms_plan`
+    - Status of the issues (open / closed)
+
+   **1.5. Quality Trainings**
+    - List of all quality trainings, compared to the :need:`wp__qms_plan`
+    - Status of the quality trainings (planned / completed)
+
+   **1.5. Process Improvement**
+    - List of all process improvements, compared to the :need:`wp__process_impr_report`
+    - Status of the process improvements (open / closed)
+
+   **1.6. External reports**
```

Comment:
How do people mean what external reports is: Do we want reports from external auditor. Or are these reports e.g. on components like for TSF.